### DISCUSSION TITLE
See Luke present some cool stuff about Quasar

### DISCUSSION THREAD

**Discussion by smolinari** [quasarframework/quasar#9357] [Announcements]
Hey everyone!

See our Luke Diebold present some news and cool stuff about Quasar to the VueJS community via the ThisDotMedia State of Vue meet-up.

https://www.youtube.com/watch?v=G-UBEjYyqjo

**Comment by shinynet** [comment]
I think it's great to see Quasar mentioned and talked about in these settings. I'm always surprised at the number of Vue devs I come across that haven't heard of Quasar

**Reply by smolinari** [reply]
Thanks. And... help spread the word. :)

Scott
